streams/src/test/java/org/apache/kafka/streams/integration/ResetIntegrationTest.java

```diff
@@ -402,6 +405,34 @@ private void cleanGlobal(final String intermediateUserTopic) {
         Assert.assertEquals(0, exitCode);
     }
 
+    private void waitUntilUserTopicsGotDeleted() {
```

Comment:
+1 on what Damian said. Currently it seems to me we're adding the same topic several times to `expectedMissingTopics`?